cask.el

```diff
@@ -730,7 +730,7 @@ If DEEP is t, all dependencies recursively will be returned."
            (-map
             (lambda (dependency)
               (list (cask-dependency-name dependency)
-                    (cask-dependency-version dependency)))
+                    (or (cask-dependency-version dependency) "0")))
```

Comment:
It's part of Emacs package spec. Dependencies without any package specified should have "0" as version.